### ISSUE TITLE
Zone de depart

### ISSUE THREAD

**Issue by ghost** [Keyzu/Pierre-Kiroule#3]
Réaliser et finaliser la zone de départ.

**Comment by ghost** [comment]
@Keyzu : Reprendre la quête de départ et faire spawn des vagues de murlocs ?

**Comment by Keyzu** [comment]
Le spawn de vague peut être sympa, à voir si ça demande pas trop de taff. 
À faire :
- Créer les 2 ou 3 quêtes restantes
- Corriger l'xp donné par la quête #2 (up 25-43 au lieu de 25-50)
- Donner plus de vie à la zone (gameobject ?)

**Comment by ghost** [comment]
Récapitulatif:
- Première quête: pnj donneur Keyzu, tuer des petits murlocs, à rendre à Keyzu
- Deuxième quête: pnj donneur Keyzu, aller ramasser l'alcool des murlocs sombre, à rendre à Keyzu
- Troisième quête: pnj donneur Keyzu, escorter Keyzu jusqu'au sommet de la zone (entrée tour), pop de murloc sur le chemin, à rendre à Keyzu
- Quatrième quête: pnj donneur Keyzu, tuer les membres de la garde rapprocher du boss, Keyzu pop au début du dernier escalier de la tour quand validé, à rendre à Keyzu
- Cinquième quête: pnj donneur Keyzu, se battre contre le boss au sommet de la tour, lock les pv du boss avant qu'il meurt, le boss fait un discourt et tp le joueur dans une nouvelle zone (? à définir) validant la quête, à rendre à Keyzu dans la nouvelle zone

**Comment by ghost** [comment]
Ça me paraît être pas mal tout ca, reste plus qu'à savoir qui va s'occuper de quoi ^^

**Comment by Keyzu** [comment]
Le script des vagues de murloc tu te sens chaud ou pas @Jybze ? :p

**Comment by ghost** [comment]
Yes ! Comment on s'organise @Keyzu ? Tu as des indications précises à me fournir ou je laisse travailler mon imagination de metteur en scène et ensuite on corrige le tir ensemble si nécessaire ?
